### PR TITLE
[RFR] Update package versions, miq-version in particular

### PR DESCRIPTION
**Pull request by mshriver**
Used freeze_all.py, so it pulled in a bunch of other stuff too.

### CODE REVIEW

File: requirements/frozen_docs.py2.txt [hunk 1]
```diff
@@ -91,7 +90,6 @@ oauthlib==2.1.0
 os-service-types==1.3.0
 oslo.i18n==3.22.1
 oslo.utils==3.37.1
-ovirt-engine-sdk-python==4.3.0
```
Comment (izapolsk) [inline requirements/frozen_docs.py2.txt]: I'm wondering if it really not necessary

Comment (izapolsk) [reply requirements/frozen_docs.py2.txt]: it is still there - https://github.com/ManageIQ/wrapanapi/blob/master/requirements.txt#L15

File: requirements/frozen_docs.py2.txt [hunk 2]
```diff
@@ -21,7 +21,6 @@ bottle-sqlite==0.1.3
 cached-property==1.5.1
 certifi==2018.10.15
 cffi==1.11.5
-cfme-testcases==0.12
```
Comment (izapolsk) [inline requirements/frozen_docs.py2.txt]: we still use this, aren't we ?

Comment (mshriver) [reply requirements/frozen_docs.py2.txt]: Correct - but its not needed for docs (or at least isn't in the docs template)

**Pull request merged**